config/tikv.toml

```diff
@@ -15,3 +15,7 @@ max-open-files = 10000
 
 [raftdb]
 max-open-files = 10000
+
+[storage]
```

Comment:
I think it's necessary to also test for API v1.

It seems that it will make codes verbose when construct `Config` in integration tests. Maybe we can write a method to detect the API version and then create the `Config` accordingly. It's OK to improve here in another PR.